forms-flow-web/src/components/FOI/FOIRequest/FOIRequest.js

```diff
@@ -1027,7 +1027,7 @@ const FOIRequest = React.memo(({ userDetail, openApplicantProfileModal }) => {
     return (
       requestState !== StateEnum.intakeinprogress.name &&
       requestState !== StateEnum.unopened.name &&
-      requestState !== StateEnum.open.name &&
+      //requestState !== StateEnum.open.name &&
       requestState !== StateEnum.appfeeowing.name &&
```

Comment:
I think we want the communication tab available in all of these states.

Reply:
Yes right! The visibility of this in other states will be handled in a different ticket story, considering any other template options that may be possible based on the state and other criteria. So for now, I made sure that this template is always visible only in states beyond 'Open'.